src/EntityComponentManager_TEST.cc

```diff
@@ -2158,6 +2158,11 @@ TEST_P(EntityComponentManagerFixture, SetEntityCreateOffset)
   manager.SetEntityCreateOffset(1000);
   Entity entity2 = manager.CreateEntity();
   EXPECT_EQ(1001u, entity2);
+
+  // Apply a lower offset, prints warning but goes through.
+  manager.SetEntityCreateOffset(500);
+  Entity entity3 = manager.CreateEntity();
+  EXPECT_EQ(501u, entity3);
```

Comment:
Did you intentionally mean to apply a lower offset to test the warning? If not, we can move this code block above the [first offset call](https://github.com/ignitionrobotics/ign-gazebo/blob/8d35373b44dbe8fa7469d949bd48b8dfd6c17d52/src/EntityComponentManager_TEST.cc#L2157-L2158), which should remove the warning.

Reply:
Yup, it was an easy addition to test coverage